config/passport.js

```diff
@@ -3,7 +3,7 @@ const JwtStrategy = require('passport-jwt').Strategy;
 const { PRIVATE_KEY } = require('../config/apiCredentials');
 
 const cookieExtractor = (request) => {
```

Comment:
I wonder if this function could be renamed. Cookie extractor sounds like it extracts the cookie, whereas we're really extracting the jwt.

Reply:
Done